src/java.base/share/classes/javax/crypto/CipherOutputStream.java

```diff
@@ -116,13 +118,14 @@ public CipherOutputStream(OutputStream os, Cipher c) {
     }
 
     /**
-     * Constructs a CipherOutputStream from an OutputStream without
-     * specifying a Cipher. This has the effect of constructing a
-     * CipherOutputStream using a NullCipher.
-     * <br>Note: if the specified output stream is null, a
+     * Constructs a <code>CipherOutputStream</code> from an
+     * <code>OutputStream</code> without specifying a <code>Cipher</code>.
+     * This has the effect of constructing a <code>CipherOutputStream</code>
+     * using a <code>NullCipher</code>.
+     * <br>Note: if the specified output stream is <code>null</code>, a
      * NullPointerException may be thrown later when it is used.
```

Comment:
Add {@code } to NullPointerException also.

Reply:
using `{@code }`